File: DevExpress-Demo/VB/GanttDemo.Wpf/Controls/GanttDemoRibbon.vb
Imports DevExpress.Xpf.Gantt
Imports System.Windows
Imports System.Windows.Controls

Namespace GanttDemo

    Public Partial Class GanttDemoRibbon
        Inherits UserControl

        Public Shared ReadOnly ExportGanttProperty As DependencyProperty = DependencyProperty.Register("ExportGantt", GetType(GanttControl), GetType(GanttDemoRibbon), New PropertyMetadata(CType(Nothing, PropertyChangedCallback)))

        Public Property ExportGantt As GanttControl
            Get
                Return CType(GetValue(ExportGanttProperty), GanttControl)
            End Get

            Set(ByVal value As GanttControl)
                SetValue(ExportGanttProperty, value)
            End Set
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub
    End Class
End Namespace

File: DevExpress-Demo/CS/GanttDemo.Wpf/Controls/GanttDemoRibbon.cs
using DevExpress.Xpf.DocumentViewer;
using DevExpress.Xpf.Gantt;
using DevExpress.Xpf.Printing;
using DevExpress.XtraPrinting;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Windows;
using System.Windows.Controls;

namespace GanttDemo {
    public partial class GanttDemoRibbon : UserControl {
       public static readonly DependencyProperty ExportGanttProperty =
            DependencyProperty.Register("ExportGantt", typeof(GanttControl), typeof(GanttDemoRibbon), new PropertyMetadata(null));
        public GanttControl ExportGantt {
            get { return (GanttControl)GetValue(ExportGanttProperty); }
            set { SetValue(ExportGanttProperty, value); }
        }
        public GanttDemoRibbon() {
            InitializeComponent();
        }
    }
}
